server/src/main/java/io/pravega/schemaregistry/service/SchemaRegistryService.java

```diff
@@ -1032,7 +1039,7 @@ private Boolean canReadChecker(SchemaInfo schema, GroupProperties prop, List<Sch
      * @return Map of group id to version that identifies the schema in the group.
      */
     public CompletableFuture<Map<String, VersionInfo>> getSchemaReferences(String namespace, SchemaInfo schemaInfo) {
-        SchemaInfo schema = normalizeSchemaBinary(schemaInfo);
+        SchemaInfo schema = normalizeSchemaBinary(schemaInfo, "", namespace);
```

Comment:
Why is groupName empty here? This could lead to an incomplete type being assigned inside `normalizeSchemaBinary`

Reply:
this method gets a map of groups to version info where the schema if it is registered. Hence this doesn't have the groupName.